sanic/asgi.py

```diff
@@ -255,142 +238,54 @@ def __init__(self) -> None:
                 instance.transport,
                 sanic_app,
             )
-
-            if sanic_app.is_request_stream:
-                is_stream_handler = sanic_app.router.is_stream_handler(
-                    instance.request
-                )
-                if is_stream_handler:
-                    instance.request.stream = StreamBuffer(
-                        sanic_app.config.REQUEST_BUFFER_QUEUE_SIZE
-                    )
-                    instance.do_stream = True
+            instance.request.stream = instance
+            instance.request_body = True  # FIXME: Use more_body?
```

Comment:
I'd advocate for this change. I originally thought there was a problem with the `read` method, until I went back and realized what this was doing in the iterator. `more_body` makes more sense to me.

Reply:
I'm not quite sure what to do with this anymore. Could you jog my memory or change it yourself?